### PR TITLE
Contrib update

### PR DESCRIPTION
**Pull request by tammojan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+platform. Issues can be reported there. If you have a patch for casacore 
+we use the github pull request system. Also to keep the casacore code
+quality high we have written down some guidelines for contributing, see below.
+
+## General considerations
+
+ * If you have problems or questions with/about git or github, first check [1].
+ * If you modify any code, make sure the test suite still runs (`make test`).
+   If it fails, fix the code or the relevant test.
+ * If you change a function/method fingerprint, update the doxygen documentation
```
Comment (mpokorny) [inline CONTRIBUTING.md]: Change `fingerprint` to `signature`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+   accordingly.
+ * If you add a function or method, add a test for it add it to the doxygen
+   documentation.
+ * The tested code coverage line count should increase, not decrease.
```
Comment (mpokorny) [inline CONTRIBUTING.md]: You might want to add a reference or hint as to how to determine the test coverage.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+## Contribution procedure
+
+ 1. Create a github account, setup SSH keys.
+ 2. Fork the casacore repository on github [3].
+ 3. Create a branch and commit your changes to this branch.
+ 4. Push your branch to your github fork (not the original casacore, you
+    probably don't have permission).
+ 5. Issue a pull request [4].
+ 6. When the pull request is reviewed and there are no problems it will be
+    accepted (peer review). Merging a pull request should always happen by
```
Comment (mpokorny) [inline CONTRIBUTING.md]: You probably don't need the `(peer review)` bit.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    someone else.
+ 7. It can happen there are some mistakes here and there, we use the github
+    commenting system to discuss these issues.
+ 8. If there is a problem with the commit you need to fix it. You can commit
```
Comment (mpokorny) [inline CONTRIBUTING.md]: Minor issue: add a comma between `commit` and `you`.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+   the subject by an empty line). See [5].
+ * If you fix an existing github issue, reference it in the commit message
+   body, e.g. 'Fixes #41'.
+ * If your Pull request does not refer to an existing github issue, it is not
+   necessary to create one (because github will present the PR just like an
+   issue).
+ * You can rewrite the history of the commits in your branch using rebase,
+   but don't rewrite the history before the first commit of your new branch.
+ * We like to keep the history clean, and prevent a lot of 'fix typo' messages.
+ * If you rewrite your history of your branch you can force push those changes
```
Comment (mpokorny) [inline CONTRIBUTING.md]: You might consider adding a hint for the appropriate git push options: `--force` or `--force-with-lease`.

**Comment by tammojan** [comment]
Thanks Martin, I've incorporated your comments. I was not aware of the `--force-with-lease` option for git, I've tried it right away here.

**Pull request merged**